docs/DocGenerator/Config.py

```diff
@@ -14,10 +14,14 @@ def __init__(self):
         self.function_regex = []
         self.ignore_paths = []
         self.valid_tags = []
-        self.required_fields = self.__fields()
-        self.ignored_fields = self.__fields()
-        with open(CONFIG_PATH) as fd:
-            self.__config_data = yaml.load(fd)
+        self.module_fields = self.__fields()
+        self.test_fields = self.__fields()
+
+        try:
+            with open(CONFIG_PATH) as fd:
+                self.__config_data = yaml.load(fd)
```

Comment:
We changed this to safe_load() because load() is deprecated.

Reply:
Fixed!